src/ralph/admin/sitetrees.py

```diff
@@ -123,6 +123,17 @@ def section(section_name, app, model):
             ]
         ),
         ralph_item(
+            title=_('Domains'),
+            url='#',
+            url_as_pattern=False,
+            perms_mode_all=False,
+            children=[
+                section(_('Domain contracts'), 'domains', 'DomainContract'),
+                section(_('Domain registrants'), 'domains', 'DomainRegistrant'),
+                section(_('Domain owners'), 'accounts', 'RalphUser'),
```

Comment:
if you mean Domain Driven Design, then not every user is domain owner (i know it could be, but only specific subset of users actually would be owning some domains), so IMO we should filter this view to show only real domain owners. What do you think?